accordion-blocks.php

```diff
@@ -103,13 +103,15 @@ public function register_block() {
 	public function enqueue_frontend_assets() {
 		$min = (defined('SCRIPT_DEBUG') && SCRIPT_DEBUG) ? '' : '.min';
 
-		wp_enqueue_script(
-			'pb-accordion-blocks-frontend-script',
-			plugins_url("js/accordion-blocks$min.js", __FILE__),
-			array('jquery'),
-			$this->plugin_version,
-			true
-		);
+		if ( has_block( 'pb/accordion-item', get_the_ID()) ) {
```

Comment:
This is great! Thank you. Would you mind reformatting to remove the spaces around the parenthesis (`(` and `)`)? I know the WordPress standard is to include spaces, but this plugin doesn't follow the WordPress standard in that regard.

The line would become:

```php
if (has_block('pb/accordion-item', get_the_ID())) {
```

Reply:
sure. I prefer PSR code style anyway